dandi/bids_validator_xs.py

```diff
@@ -711,6 +711,19 @@ def validate_bids(
         regex_schema,
         debug=debug,
     )
+    # Record schema version.
+    # Not sure whether to incorporate in validation_result.
+    if bids_schema_dir == os.path.join(
+        os.path.abspath(os.path.dirname(__file__)),
+        "data",
+        "schema",
+    ):
+        # Declare we are using live version,
+        # string will evaluate as larger than numbered versions.
+        schema_version = "live"
```

Comment:
oh well, semver would puke on it quite loud happen someone decide to use that:

```
$> python3 -c 'from semantic_version import Version; print(Version("live.0.0")>Version("1.7.0+012"))'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/semantic_version/base.py", line 105, in __init__
    major, minor, patch, prerelease, build = self.parse(version_string, partial)
  File "/usr/lib/python3/dist-packages/semantic_version/base.py", line 311, in parse
    raise ValueError('Invalid version string: %r' % version_string)
ValueError: Invalid version string: 'live.0.0'
1 24249 ->1.....................................:Wed 11 May 2022 04:25:08 PM EDT:.
lena:~
$> python3 -c 'from semantic_version import Version; print(Version("live")>Version("1.7.0+012"))' 
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/semantic_version/base.py", line 105, in __init__
    major, minor, patch, prerelease, build = self.parse(version_string, partial)
  File "/usr/lib/python3/dist-packages/semantic_version/base.py", line 311, in parse
    raise ValueError('Invalid version string: %r' % version_string)
ValueError: Invalid version string: 'live'
```

since I still hope that eventually we arrive at proper semver comparison, let's do indeed some rudiculous large number as you had it but semver friendly:

```suggestion
        schema_version = "99999.0.0"
```

and semver would be happy too and it would work until we retire.  I will accept my invaluable contribution right away to see if tests survive